index.html

```diff
@@ -516,6 +517,111 @@ <h2>Algorithms</h2>
 by default, and abort processing upon detection.
       </p>
 
+      <section>
+        <h3>Suite Selection Algorithm</h3>
+
+        <p>
+This algorithm is used to configure a cryptographic suite to be used by the
+<a href="https://www.w3.org/TR/vc-data-integrity/#add-proof">Add Proof</a> and
+<a href="https://www.w3.org/TR/vc-data-integrity/#verify-proof">Verify Proof</a>
+functions in [[[VC-DATA-INTEGRITY]]]. The algorithm takes an options object
+([=map=] |options|) as input and returns a cryptosuite
+([=struct=] |cryptosuite|).
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Initialize |cryptosuite| to an empty [=struct=].
+          </li>
+          <li>
+If |options|.|type| is `DataIntegrityProof` and |options|.|cryptosuite| is
+`ecdsa-rdfc-2019` then:
+            <ol class="algorithm">
+              <li>
```

Comment:
I applied the markup and the CSS conflicted with some stuff that's going on in the local spec wrt. appropriate numbering for algorithms (the indention and the numbering got messed up). We've found it useful for implementers to be able to speak to step an substep numbers using the numbers we provide in the spec. 

I'll try to do this in a future edit, after I get the CSS issues sorted (I have a deep work queue today, so have to move on to other things and need to get the normative text in this PR merged before getting to other PRs).

There is also an issue w/ variable highlighting... I'd like to be able to define the variables in the paragraph above and have the highlighting continue in the algorithm below, but it looks like ReSpec has change the way the highlighting works and the variable highlighting is now broken. I'll need to look into the ReSpec code to see if what I'm trying to do is possible (be able to specify an entire section including paragraph tags and algorithm classes as all in the same scope. If not, I'll have to figure out how to modify the numbering CSS we're using in order to not conflict w/ the algorithm numbering.

All that to say, skipping this (good) suggestion for now because I've spent too much time fighting w/ the tooling today (broken refs in reffy, broken CSS for "switch" and "algorithm").